ui/App/ProfileScreen.svelte

```diff
@@ -125,16 +137,73 @@
   }
 
   let showRequests: boolean = false;
-
   showNotificationsForFailedProjects();
+
+  const wallet = svelteStore.get(Wallet.store);
+  let state: "loading" | "loaded" | "error" = "loading";
+
+  async function loadSidebarData(): Promise<void> {
+    const address = session.identity.metadata.ethereum.address;
+    state = "loading";
+    try {
+      const gnosisSafeWallets = await Safe.getSafesByOwner(
+        wallet.environment,
+        address
+      );
+      ownedOrgs = await graph.getOwnedOrgs([address, ...gnosisSafeWallets]);
+      const ensName = await wallet.provider.lookupAddress(address);
+      registration = await getRegistration(ensName);
+      state = "loaded";
+    } catch (err: unknown) {
+      error.show(
+        new error.Error({
+          code: error.Code.ProjectRequestFailure,
+          message: "Failed to fetch ENS registration",
+          source: err,
+        })
+      );
+      state = "error";
+    }
+  }
+
+  loadSidebarData();
+
+  const showSidebar: boolean =
```

Comment:
We should make this reactive, so the screen updates without a manual refresh if the wallet state changes.

Reply:
What does this mean? `$: loadSidebarData()`?